stdlib/dbm/sqlite3.pyi

```diff
@@ -0,0 +1,28 @@
+from _typeshed import ReadableBuffer, StrOrBytesPath, Unused
+from collections.abc import MutableMapping
+from typing import Final, Literal, LiteralString, Self, TypeAlias
+
+BUILD_TABLE: Final[LiteralString]
+GET_SIZE: Final[LiteralString]
```

Comment:
Could also include the full string literals - but don't think it makes sense to store raw SQL in stubs.

Reply:
Interesting question: Is `Final[LiteralString] == Final[str]`?

Reply:
@srittau no. https://pyright-play.net/?strict=true&code=GYJw9gtgBALgngBwJYDsDmUkQWEMoBiqAhgDYA0UAMkjAKYhkDKMIqaAUBwCZ3BRo6MAPoBnSHTGsAFAEooAWgB8UUawBcHKNqgghAVxApVMoijKyuAD3WESpANpqQAXSgBeAULESpIORxwtmZkDjT0jKQsbOhunoIi4hCSzgFAA

I think `Final[LiteralString]` is a more precise type here because it will allow these constants to get used in a place where a LiteralString is expected.

Reply:
Yes, it's useful to enforce `LiteralString` for DB cursors to avoid SQL injection (I do this in a project of mine), so it's helpful to have these as literal strings.